web-ts/src/components/__tests__/__snapshots__/CategoriesToolbar.spec.tsx.snap

```diff
@@ -6,7 +6,7 @@ exports[`CategoriesToolbar should render Toolbar, if category can be found 1`] =
   viewportSmall={true}
 >
   <ToolbarItem
-    href="https://cms-test.integreat-app.de/augsburg/de/wp-json/ig-mpdf/v1/pdf?url=%2Faugsburg%2Fde%2Fwillkommen"
+    href="https://cms-test.integreat-app.de/augsburg/de/wp-json/ig-mpdf/v1/pdf"
```

Comment:
Are we sure this change is fine?

Reply:
Used just the wrong category for the test. Fixed.